src/common/transformations/src/transformations/op_conversions/group_normalization_decomposition.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/group_normalization_decomposition.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/opsets/opset12.hpp"
```

Comment:
could you delete this include?

Reply:
Is it preferred to include every op separately?

Reply:
yes, this include adds ~5mb to each obj file size, the sum of all obj file sizes exceeds 4 gb limit on Windows in Debug configuration, so .lib file fails to create.

Reply:
Ok, done